website/.gitignore

```diff
@@ -0,0 +1,59 @@
+### Node ###
```

Comment:
Nit: Where is this from? This file seems way too big many of these lines are completely irrelevant. 

No real harm but rather not confuse users with unrelated stuff.

Reply:
for this and for your other comment on license - 

my process was:
- npm init -y (which set the license to ISC)
- yarn add ocular-gastby@latest gatsby@latest react@latest react-dom@latest
- ocular init
- some touch-ups of ocular-config.js and package.json based on loaders.gl

i suspect the .gitignore is generated by npm init.